packages/ui-popover/src/Popover/index.tsx

```diff
@@ -570,7 +576,17 @@ class Popover extends Component<PopoverProps, PopoverState> {
               : placement
         } as Partial<ContextViewProps> & { ref: any }
 
-        return <ContextView {...viewProps}>{content}</ContextView>
+        return (
+          <ContextView
+            {...viewProps}
+            themeOverride={{
```

Comment:
we shouldn't use `themeOverride` for our own components

Reply:
removed